evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -887,25 +887,28 @@ def test_partial_dependence_datetime(
 
     X = pd.DataFrame(X, columns=[str(i) for i in range(X.shape[1])])
     y = pd.Series(y)
-    X["dt_column"] = pd.Series(pd.date_range("20200101", periods=X.shape[0]))
-
+    random_dates = pd.Series(pd.date_range("20200101", periods=10)).sample(
```

Comment:
In order to speed up these tests, I used fewer unique dates. I think this points to how our approach for handling dates does not scale super well. I think we should revisit this in the near future.

Reply:
Good to file an issue? :)

Reply:
Will do!